specs/core/0_beacon-chain.md

```diff
@@ -17,19 +17,28 @@
         - [Time parameters](#time-parameters)
         - [Reward and penalty quotients](#reward-and-penalty-quotients)
         - [Status codes](#status-codes)
-        - [Special record types](#special-record-types)
+        - [Max operations per block](#max-operations-per-block)
         - [Validator registry delta flags](#validator-registry-delta-flags)
         - [Signature domains](#signature-domains)
     - [Data structures](#data-structures)
-        - [Deposits](#deposits)
-            - [`DepositParametersRecord`](#depositparametersrecord)
+        - [Beacon chain operations](#beacon-chain-operations)
```

Comment:
Not sure if the following categories are very accurate.
1. `AttestationData` is also used in `CasperVotes`.
2. If we put all `*Data` class to here, Why `ProposalSignedData` is still in `ProposalSignedData` section?